src/merkle/path_set.rs

```diff
@@ -202,8 +213,8 @@ impl MerklePathSet {
     }
 
     /// Returns hash of the root
-    fn compute_path_root(path: &[Word], depth: u32, index: u64) -> Word {
-        let mut pos = 2u64.pow(depth) + index;
+    fn compute_path_root(path: &[Word], node: NodeIndex) -> Word {
```

Comment:
nit: should the parameter be named `index` rather than `node`?

Reply:
Done!